dashboard/config/routes.rb

```diff
@@ -81,7 +81,11 @@ module OPS
   root :to => "home#index"
   get '/home_insert', to: 'home#home_insert'
   get '/health_check', to: 'home#health_check'
-  get '/home/:action', controller: 'home'
+  namespace :home do
+    HomeController.instance_methods(false).each do |action|
+      get action, action: action
+    end
+  end
```

Comment:
`:action` and `:controller` magic paths are deprecated in Rails 5 due to security issues, so the new code explicitly reproduces the same behavior by looking at the public instance methods in the class and declaring individual routes for each method.